src/main/rp/RP.ts

```diff
@@ -18,10 +20,13 @@ import {
   SupportedVersion,
   VerifiedAuthenticationResponse,
 } from '../types';
+import { AuthorizationEvents } from '../types/Events';
 
 import Builder from './Builder';
 import { createRequestOptsFromBuilderOrExistingOpts, createVerifyResponseOptsFromBuilderOrExistingOpts, isTargetOrNoTargets } from './Opts';
 
+export const rpEventEmitter = new EventEmitter();
```

Comment:
Move to RP so we have an emitter per RP (you can have multiple RPs if you want)

Reply:
moved